ai-ml/emr-spark-rapids/addons.tf

```diff
@@ -52,7 +52,7 @@ module "eks_blueprints_addons" {
   # CoreDNS Autoscaler helps to scale for large EKS Clusters
   #   Further tuning for CoreDNS is to leverage NodeLocal DNSCache -> https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/
   #---------------------------------------------------------------
-  enable_cluster_proportional_autoscaler = true
+  enable_cluster_proportional_autoscaler = false
```

Comment:
Please remove this add-on as its not required anymore. Its built into CoreDNS add-on.